comms/dht/src/config.rs

```diff
@@ -151,6 +156,7 @@ impl Default for DhtConfig {
             saf_max_message_size: 512 * 1024,
             dedup_cache_capacity: 2_500,
             dedup_cache_trim_interval: Duration::from_secs(5 * 60),
+            dedup_allowed_message_occurrences: 1,
```

Comment:
**NIT:** We may need a TODO comment here to test or further implement this feature.